src/applications/vaos/new-appointment/components/TypeOfMentalHealthPage.unit.spec.jsx

```diff
@@ -42,38 +41,26 @@ describe('VAOS Page: TypeOfMentalHealthPage', () => {
     );
     await screen.findByText(/Continue/i);
 
-    // Then the primary header should have focus
-    const radioSelector = screen.container.querySelector('va-radio');
-    expect(radioSelector).to.exist;
-    expect(radioSelector).to.have.attribute(
-      'label',
-      'Which type of mental health care do you need?',
-    );
+    // Should show title
+    expect(
+      await screen.findByRole('heading', {
+        level: 1,
+        name: /Which type of mental health care do you need\?/,
+      }),
+    ).to.exist;
 
-    // And the user should see radio buttons for each clinic
-    const radioOptions = screen.container.querySelectorAll('va-radio-option');
+    // And the user should see radio buttons for each type of sleep care
+    const radioOptions = screen.getAllByRole('radio');
     expect(radioOptions).to.have.lengthOf(2);
-    expect(radioOptions[0]).to.have.attribute(
-      'label',
-      'Mental health services',
-    );
-    expect(radioOptions[1]).to.have.attribute(
-      'label',
-      'Substance use problem services',
-    );
+    await screen.findByLabelText(/Mental health services/i);
+    await screen.findByLabelText(/Substance use problem services/i);
 
     fireEvent.click(screen.getByText(/Continue/));
     // Then there should be a validation error
-    // Assertion currently disabled due to
```

Comment:
One benefit of using a custom non-shadow DOM widget is that we can test the validation assertions in unit tests.